generators/firechip/src/main/scala/BridgeBinders.scala

```diff
@@ -55,19 +56,17 @@ class WithFASEDBridge extends RegisterBridgeBinder({
         val nastiKey = NastiParameters(axi4Bundle.r.bits.data.getWidth,
                                        axi4Bundle.ar.bits.addr.getWidth,
                                        axi4Bundle.ar.bits.id.getWidth)
-        FASEDBridge(axi4Bundle, t.reset.toBool,
+        FASEDBridge(t.clock, axi4Bundle, t.reset.toBool,
           CompleteConfig(p(firesim.configs.MemModelKey), nastiKey, Some(AXI4EdgeSummary(edge))))
       })
     }).toSeq
 })
 
-class WithTracerVBridge extends RegisterBridgeBinder({
-  case target: HasTraceIOImp => TracerVBridge(target.traceIO)(target.p)
-})
+class WithTracerVBridge extends Config((_,_,_) => { case InstantiateTracerVBridges => true })
```

Comment:
I'm curious why this moves away from the IOBinders approach? 
I believe @abejgonzalez is developing the Dromajo widget as another BridgeBinder that attaches to `HasTraceIOImp`.

Reply:
I made this change prior to your generator unification work, and i thought it would be nice to move the instantiation closer to the source.  I was thiking that ideally the bridge could be instantiated directly in the tile itself perhaps using one of the generation hooks that exists in rocketchip. Then i could just delete the target mixins altogether.

But yes, in light of recent changes, I think it makes sense to continue plumbing this out to the top. I'll switch back to that scheme.

Reply:
Yes. Jerry is right. Right now the Dromajo Bridge nicely hooks into the `TraceIO`.